client/src/scss/gallery.scss

```diff
@@ -57,3 +16,22 @@ div.controls button {
 .v-card * {
   height: 100%;
 }
+
+// Make text smaller in the data browser.
+table.v-table tbody td {
+  font-size: 9px;
```

Comment:
We should probable use `rem` rather than absolute pixel values.

Reply:
👍